specification/common/attribute-precedence.md

```diff
@@ -0,0 +1,175 @@
+# Attribute Precedence on transformation to non-OTLP formats
+
+**Status**: [Experimental](../document-status.md)
+
+<details>
+<summary>Table of Contents</summary>
+
+<!-- toc -->
+
+- [Overview](#overview)
+- [Attribute hierarchy in the OTLP message](#attribute-hierarchy-in-the-otlp-message)
+- [Precedence per Signal](#precedence-per-signal)
+  - [Traces](#traces)
+    - [Span Events](#span-events)
+    - [Span links](#span-links)
+  - [Metrics](#metrics)
+    - [Metric exemplars](#metric-exemplars)
+  - [Logs](#logs)
+- [Considerations](#considerations)
+- [Example](#example)
+- [Useful links](#useful-links)
+
+<!-- tocstop -->
+
+</details>
+
+## Overview
+
+This document provides supplementary guidelines for the attribute precedence 
+that exporters should follow when translating from the hierarchical OTLP format
+to non-hierarchical formats.
+
+A mapping is required when flattening out attributes from the structured OTLP
+format, which has attributes at different levels (e.g., Resource attributes, 
+InstrumentationScope attributes, attributes on Spans/Metrics/Logs) to a
+non-hierarchical representation (e.g., OpenMetrics labels).
+In the case of OpenMetrics, the set of labels is completely flat and must have 
+unique labels only 
+(https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#labelset).
+Since OpenTelemetry allows for different levels of attributes, it is feasible
+that the same attribute appears multiple times on different levels.
+
+This document aims to provide guidance on how OpenTelemetry attributes can be 
+consistently mapped to flat sets.
+
+## Attribute hierarchy in OTLP messages
+
+Since the OTLP format is a hierarchical format, there is an inherent order in 
+the attributes.
+In this document, 
+[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
+attributes are considered to be at the top of the hierarchy, since they are the
+most general attributes. 
+Attributes on individual Spans/Metric data points/Logs are at the bottom of the
+hierarchy, as they are most specialized and only apply to a subset of all data.
+
+**A more specialized attribute that shares an attribute key with more general 
+attribute will take precedence.** 
+
+In some cases it might be desirable to overwrite an attribute like this.
```

Comment:
I think it may be possible to remove this (and the example) and change the above to: 

```
**A more specialized attribute that shares an attribute key with more general 
attribute will take precedence and overwrite the more general attribute.**
```

Reply:
Nice! Still thinking about an example...